packages/tools/sysutils/jslisten/profile.d/98-jslisten

```diff
@@ -51,7 +51,23 @@ button1=${BTN_BACK60S}
 
 EOF
 		fi
+		if [[ "${HW_DEVICE}" =~ RG351P ]]
+    			then
+		cat <<EOF >>${JSLISTENCONF}
+# L3 + R1
+[BrightnessUp]
+program="/usr/bin/brightness up"
+button1=8
```

Comment:
We've been working to remove hard coded values from configuration files wherever we can.